Formula/makensis.rb

```diff
@@ -32,8 +39,13 @@ def install
     # https://sourceforge.net/p/nsis/bugs/1085/
     ENV.libstdcxx if ENV.compiler == :clang
 
+    # requires zlib (win32) to build utils
+    resource("zlib-win32").stage do
```

Comment:
Weird; for some reason the `include`, `lib`, and `test` directories in zlib-win32 have permissions set to 555 instead of 755.  We should be able to fix that when unpacking the resource.